### ISSUE TITLE
[0.26][Andr13] Issue connecting when gapps are installed

### ISSUE THREAD

**Issue by Lelaxlp** [AsteroidOS/AsteroidOSSync#186]
This is an issue with my rom installation, so if you don't wont to look into this, feel free to delete it.
I ran into this super weired issue:

LOS20/Android13 with NikGapps Core: Finds watch, doesn't connect/ sync; watch never shows any hint that it is being connected to.
LOS20/Android13 without NikGapps: Works just fine with the watch.

Conclusion: Google somehow messes with my nearby devices permissions;
any idea what I could do?
- I do grant the permisson nearby devices when asked for it during first launch of the app.
- The Google Wear OS App worked to connect the watch (had to install it to unlock bootloader).
- Always allow location in background is not available to set for AsteroidOSSync, could this be the issue?
- I have it rooted with magisk and installed a module, Viper4Android, could root be the problem?
- What logs should I provide?

LG, Lelax

**Comment by jrtberlin** [comment]
Sorry to hear that you are having issues.

I tried to reproduce the issue you are experiencing:
- It works for me on a phone without gapps.
- It didn't work initially on a phone with gapps.
  - It worked after I enabled the "Synchronization Service" notification channel in Androids settings. It kept working after I disabled it. I don't think that's it tbh, but I can't reproduce it since, even with reinstalling the app.

Did you install the app via F-Droid, NeoStore or manually? 

## Things you can try:
- Can you check if you have all notification types of the app enabled?
- Stopping the app from the app info page fully and try to pair again
- Check that the watch is not paired in Androids Bluetooth settings.
- Clearing App Data/App Cache
- Please check in the Android settings if all permissions are granted. If they are not, please report back which ones are disabled.

**Comment by Lelaxlp** [comment]
Thanks for your reply!
What I now did:
- App is installed via F-Droid
- Reinstalled app multiple times
- Made sure all possible permissions are granted (I was only able to grant those I see, can I maybe grant all of the permissions via adb to fix this?)
- Switched the "Synchronization Service" notification channel multiple times while (re)starting the app
- The watch is not paired in the Bluetooth settings (Pre-AsteroidOS watch with maybe the same bluetooth adress is also deleted)
- Cleared the Data/ App cache multiple times while stopping the app
- Watch is found, but never makes pairing request/ never syncs
Again, this more of a problem of my rom/ gapps, so feel free to close this if there is no solution.
LG, Lelax

**Comment by Lelaxlp** [comment]
Update:
Super weired things are happening:
- The Issue is my rom. I wiped my phone and it still didn't work, neither stock nor with gapps.
- Now having installed LOS 19.1 aka Android 12L, it works perfectly fine.
- The S7 with LOS 20 (no gapps) worked just fine, despite the rom being from the same dev, so I assume they both are using the same "build tree".

Conclusion: There are weired issues with permissions in Android 13, and maybe this app can do something to make it work, but for now I will report this as an issue with the rom and not this app.

TL;DR:
Stock LOS 20 fails to connect on one phone but works on another running stock LOS 20,
only solution was to downgrade to LOS 19.1
